### DISCUSSION TITLE
Subject: Request for Keyboard Shortcuts in Obsidian Web Chrome Extension

### DISCUSSION THREAD

**Discussion by bravenewworld314** [coddingtonbear/obsidian-web#103] [Ideas]
Hello,  I'm a user of the Obsidian Web Chrome extension, and I find it to be an incredibly useful tool for my daily note-taking needs. However, I've noticed that the extension currently doesn't have support for keyboard shortcuts, or if it does, they're not well documented.  Keyboard shortcuts can dramatically speed up the note-taking process and improve the overall user experience. I've used the Notion Web Clipper extension as an example, which provides a very intuitive and efficient keyboard shortcut system.  Is it possible to add this feature to the Obsidian Web Chrome extension, or if it's already present, could you please provide some documentation on how to use it? I believe this would greatly benefit all users of the extension.  Thank you for your time and for the wonderful tool you've created.

**Comment by coddingtonbear** [comment]
This sort of functionality was added as part of v3 of this plugin some weeks back.  The default key binding is `Alt+Shift+O`, but you can see your currently-selected keybinding and instructions about how to change it from the settings screen.
